### PR TITLE
New texts sep28

### PR DESCRIPTION
**Pull request by manishjain999**
More text changes to be Merged. Please review.

### CODE REVIEW

File: config/config-es.json [hunk 1]
```diff
@@ -494,6 +503,7 @@
         "buttonLogoutText": "Cerrar Sesión",
         "registeredVersionText": "Suscripción activa",
         "logoutAlertText": "¿Está seguro de que deseas cerrar la sesión?",
+        "titleUserIdText": "Identificación de usuario:",
```
Comment (jeznag) [inline config/config-es.json:506]: duplicate?

File: config/config.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
         "emailId":"team@focusbear.io",
         "unlockMachineButtonText":"Unlock the machine",
         "wrapupButtonText":"Let's wrap up for the day",
-        "stayFocusedText":"OK - Stay focused throughout the day!",
+        "stayFocusedText":"Now let's set up an account for you",
```
Comment (jeznag) [inline config/config.json:34]: isn't this shown at the end of the morning routine? Don't we need a separate translation key?

File: config/config.json [hunk 3]
```diff
@@ -74,7 +74,15 @@
         "habitsHardReasonText": "My habits are too hard - I can't handle them",
         "emailTitle": "Your Email (if you want us to reply):",
         "buttonPostponeText": "Postpone",
-        "noPurchasesFoundText": "No purchases found."
+        "noPurchasesFoundText": "No purchases found.",
+        "someProblemInCredentialText": "There is some problem. Please Login again.",
```
Comment (jeznag) [inline config/config.json:78]: There was a problem connecting to the backend server. Please login again.

File: config/config-es.json (hunk 1 above)
Comment (manishjain999) [reply config/config-es.json:506]: I deleted the duplicate one in English..

Comment (manishjain999) [reply config/config-es.json:506]: ES need to delete the one without , at end. 507

File: config/config.json (hunk 2 above)
Comment (manishjain999) [reply config/config.json:34]: "stayFocusedText":"Now let's set up an account for you", is added new and needs a translation for ES

File: config/config.json [hunk 4]
```diff
@@ -530,7 +539,7 @@
         "subscriptionExpiredText":"Your subscription has expired"
     },
     "onboardingPostponeVcInfo": {
-        "title": "If you have something urgent, you can postpone the Onboarding process for a few minutes.",
+        "title": "If you have something urgent, you can postpone the Onboarding process for few Minutes.",
```
[the remark this answers is not in the document]
Comment (manishjain999) [reply config/config.json:542]: ok

**Pull request merged**